lib/Getopt/Std.pm

```diff
@@ -119,6 +122,7 @@ sub getopt (;$$) {
 	        $$hash{$first} = $rest;
 	    }
 	    else {
+            no strict 'refs';
```

Comment:
Indentation doesn't quite match the following lines.

Reply:
Welcome to the already existing world of Mixed tabs/spaces. Can I put you down as a yes vote for doing an eventual mass perltidy of blead? 😛

Reply:
I wouldn't go as far as mass perltidy, but making indentation consistent as an isolated commit would be useful IMO